pyglider/seaexplorer.py

```diff
@@ -111,6 +111,8 @@ def raw_to_rawnc(indir, outdir, deploymentyaml, incremental=True, min_samples_in
                                             dayfirst=True)
                     # If AD2CP data present, convert the timestamps to datetime type
                     if 'AD2CP_TIME' in out.columns:
+                        # Set datestamps with date 00000 to None to prevent datetime error
+                        out.loc[out.AD2CP_TIME.str[:6]=='000000','AD2CP_TIME'] = None
```

Comment:
I missed that we had picked up a pandas dependency - is pandas really necessary here?  I'm also not clear why time is being saved as seconds since 1970 instead of datetime64 below?  I suspect this is historical in your adcp processing, but I'm not sure it is necessary.

Reply:
We already had pandas in `raw_to_rawnc` to read in the csv input files. The data at this point in processing is a dataframe already so I thought it made sense to use

Reply:
ha, shows you what I remember about my own code.  OK, that seems fine.

Reply:
the part with saving to seconds since 1970 also appears to be original

Reply:
We could change it though, certainly

Reply:
No lets leave it for now.  Certainly not the issue here.

Reply:
We've all been there! the time could use a look at though. I guess that could be done as part of the the move to OG 1.0 once it's finalised